src/autogluon/cloud/version.py

```diff
@@ -1,2 +0,0 @@
-"""This is the autogluon version file."""
-__version__ = '0.1b20221111'
```

Comment:
This file should be in `.gitignore` because it is auto-generated.

Reply:
Yea, this is deleting it. I've added it to `.gitignore`